### PR TITLE
Escape html of content on the task listing tab

### PR DESCRIPTION
**Pull request by maethu**
This PR fixes two issue:
- breadcrumb part of the title
- containing dossier title

### CODE REVIEW

File: docs/HISTORY.txt [hunk 1]
```diff
@@ -5,6 +5,8 @@ Changelog
 2017.3.0 (unreleased)
 ---------------------
 
+- CSRF:Escape html for Dossier title on task listing. [mathias.leimgruber]
```
Comment (phgross) [inline docs/HISTORY.txt]: `CSRF:` --> `XSS:`

File: docs/HISTORY.txt [hunk 2]
```diff
@@ -5,6 +5,8 @@ Changelog
...
+- CSRF: Escape html for the breadcrumbs part for get_link on tasks. [mathias.leimgruber]
```
Comment (phgross) [inline docs/HISTORY.txt]: `CSRF:` --> `XSS:`

File: docs/HISTORY.txt (hunk 1 above)
Comment (maethu) [inline docs/HISTORY.txt]: done

File: docs/HISTORY.txt (hunk 2 above)
Comment (maethu) [inline docs/HISTORY.txt]: done

**Comment by maethu** [comment]
@phgross  Yes it's escaped: https://github.com/4teamwork/opengever.core/blob/cc1e8300a8242bcbf570411aee5ecc06ff649d5f/opengever/globalindex/model/task.py#L286

**Comment by maethu** [comment]
@phgross updated

**Pull request merged**